### PR TITLE
Js1

### PR DESCRIPTION
**Pull request by OleksiiTarasenko**
Lesson 24 & 25

### CODE REVIEW

File: ba/js/lesson1/dist/script.dev.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+//Task 1
+var yourName = prompt("Your name..");
+console.log("Hello " + yourName + " !"); //Task 2
+
+var CURRENT_YEAR = 2020;
+var birthYear = prompt("When your were born?   ");
```
Comment (idovbush) [inline ba/js/lesson1/dist/script.dev.js]: use let instead of var
When were you born*

File: ba/js/lesson1/dist/script.dev.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+var age = CURRENT_YEAR - birthYear;
+console.log("You are " + age + " years old!"); //Task 3
+
+var side = prompt("Enter length of side of square");
+console.log("The perimetr equals: ".concat(side * 4)); //Task 4
```
Comment (idovbush) [inline ba/js/lesson1/dist/script.dev.js]: you can use it but here it's not okay, just `The perimeter equals: ${side} * 4`. But whether it it for practice it's ok

File: ba/js/lesson1/dist/script.dev.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+var radius = prompt("Enter radius of the circle");
+console.log("The area of circle: " + Math.pow(radius, 2) * 3.14); //Task 5
```
Comment (idovbush) [inline ba/js/lesson1/dist/script.dev.js]: use string literals instead of "" + ""

File: ba/js/lesson1/dist/script.dev.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+alert(`hello ${1}`); // ?   'hello 1'
+
+alert(`hello ${"name"}`); // ?  'hello name'
+
+alert(`hello ${name}`); // ?  'hello Ilya
+*/
+//Extra 3
+
+var a = prompt("Первое число?", 1);
```
Comment (idovbush) [inline ba/js/lesson1/dist/script.dev.js]: let a = +prompt("Первое число?", 1);
let a = +prompt("Первое число?", 2);
alert(a + b);

File: ba/js/lesson1/dist/script.dev.js (hunk 1 above)
Comment (OleksiiTarasenko) [reply ba/js/lesson1/dist/script.dev.js]: Please do not  take to your consideration everything in file    /dist/script.dev.js
This file was automatically generated by CompileHero, not by me
My work is in script.js

File: ba/js/lesson2/script.js [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+    break;
+  case "8":
+    console.log("*");
+    break;
+  case "9":
+    console.log(")");
+    break;
+  default:
+    console.log("Wrong button!!!");
+    break;
```
Comment (idovbush) [inline ba/js/lesson2/script.js]: we do not need break here

**Pull request merged**